vllm/assets/monitors/token_throughput.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": 2,
+    "created_at": "2024-06-26",
+    "last_updated_at": "2024-06-26",
+    "title": "vLLM application token usage is high",
+    "description": "This monitor will alert you if vLLM is generating a lot of tokens.",
```

Comment:
Can you add more context on this? Use the description to:
- Define the problem stated by the title.
- Answer why this is an issue worth alerting on.
- Describe the impact of the problem.

Reply:
Thank you very much for this @estherk15!! I've tried to address it, lmk